tssc/step_implementers/push_container_image/skopeo.py

```diff
@@ -14,6 +14,9 @@
                                              automatically be applied
 | `src-tls-verify`  | True      | `'true'` | Whether to very TLS for source of image
 | `dest-tls-verify` | True      | `'true'` | Whether to verify TLS for destination of image
+| `containers-config-auth-file` | True | `'~/.skopeo-auth.json'` | \
```

Comment:
Formatting appears to have been thrown off here as well

Reply:
it generates correctly. i just didn't want to re-build all the table for the longer variable and then make all the descriptions have like 10 chars to fit in.